src/test/regress/sql/multi_extension.sql

```diff
@@ -395,6 +395,14 @@ SELECT * FROM multi_extension.print_extension_changes();
 ALTER EXTENSION citus UPDATE TO '10.2-4';
 SELECT * FROM multi_extension.print_extension_changes();
 
+-- Snapshot of state at 10.2-5
+ALTER EXTENSION citus UPDATE TO '10.2-5';
+SELECT * FROM multi_extension.print_extension_changes();
+
+-- Test downgrade to 10.2-4 from 10.2-5
+ALTER EXTENSION citus UPDATE TO '10.2-4';
+ALTER EXTENSION citus UPDATE TO '10.2-5';
+
```

Comment:
I like these lines. However some of the next tests also need to be updated now. Please go over the occurrences of `10.2-4` later in the file and change them into `10.2-5` if it makes sense to you.

Some examples:

> -- Test downgrade to 10.2-4 from 11.0-1

> ALTER EXTENSION citus UPDATE TO '10.2-4';